WordPress/src/main/java/org/wordpress/android/widgets/DiffView.kt

```diff
@@ -18,10 +18,14 @@ class DiffView : AppCompatTextView {
     constructor(context: Context, attrs: AttributeSet) : super(context, attrs)
     constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
 
-    fun showDiffs(diffs: List<Diff>) {
+    fun showDiffs(diffs: List<Diff>, trimNewline: Boolean = false) {
         text = null
 
         diffs.forEachIndexed { index, diff ->
+            if (trimNewline && index == diffs.size - 1 && diff.value.equals("\n")) {
```

Comment:
After some testing, looks like there are some instances where the newline is not the only element in the trailing diff, so we might want to use something like `trimEnd('\n')` on the last diff value.

Reply:
I wasn't able to reproduce a `Diff` with anything other than a single `\n` character, but I updated the `DiffView.showDiffs` method in ab20d08.